dashboard/public/css/main.css

```diff
@@ -105,3 +105,53 @@
         background-color: inherit;
     }
 }
+
+meter {
+    margin: -4em auto 1em;
+    width: 100%;
+    background: none;
+
+}
+
+meter::-webkit-meter-bar {
+    background: none;
+}
+
+
+meter[value="1"]::-webkit-meter-optimum-value {
+    background: rgb(255, 0, 0);
+}
+
+meter[value="2"]::-webkit-meter-optimum-value {
+    background: yellow;
+}
+
+meter[value="3"]::-webkit-meter-optimum-value {
+    background: orange;
+}
+
+meter[value="4"]::-webkit-meter-optimum-value {
+    background: green;
+}
+
+meter[value="1"]::-moz-meter-bar {
+    background: #ca1a64;;
+}
+
+meter[value="2"]::-moz-meter-bar {
+    background: yellow;
+}
+
+meter[value="3"]::-moz-meter-bar {
+    background: orange;
+}
+
+meter[value="4"]::-moz-meter-bar {
+    background: green;
+}
+
+#password-strength-text {
+    margin: -1.3em auto 0em;
+    width: 100%;
+    background: none;
+}
```

Comment:
- No newline at end of the file.
- Move the additional styling before the media query (`@media (max-width: 991px)`)

Reply:
No newline at end of the file.